pal/src/thread/thread.cpp

```diff
@@ -2786,12 +2786,12 @@ int CorUnix::CThreadMachExceptionHandlers::GetIndexOfHandler(exception_mask_t bm
 
 #endif // HAVE_MACH_EXCEPTIONS
 
-int GetCurrentThreadStackBounds(char** stackBase, char** stackEnd)
+void GetCurrentThreadStackLimits(ULONG_PTR* lowLimit, ULONG_PTR* highLimit)
 {
 #ifdef _TARGET_MAC64
     // This is a Mac specific method
-    *stackBase = (char*)pthread_get_stackaddr_np(pthread_self());
-    *stackEnd = (char*)(stackBase - pthread_get_stacksize_np(pthread_self()));
+    *highLimit = (ULONG_PTR)pthread_get_stackaddr_np(pthread_self());
+    *lowLimit = (ULONG_PTR)(stackBase - pthread_get_stacksize_np(pthread_self()));
```

Comment:
why `lowLimit` and `highLimit`? The order of the arguments are also changed? i.e. since there is no stackBase local parameter now, what stackBase at 2794 needs to be?